ocp-metadata/ocp-metadata.go

```diff
@@ -96,12 +96,17 @@ func (meta *Metadata) GetCurrentPodCount() (int, error) {
 	if err != nil {
 		return podCount, err
 	}
-	for _, node := range nodeList.Items {
-		podList, err := meta.clientSet.CoreV1().Pods(metav1.NamespaceAll).List(context.TODO(), metav1.ListOptions{FieldSelector: "status.phase=Running,spec.nodeName=" + node.Name})
-		if err != nil {
-			return podCount, err
+	podList, err := meta.clientSet.CoreV1().Pods(metav1.NamespaceAll).List(context.TODO(), metav1.ListOptions{FieldSelector: "status.phase=Running"})
```

Comment:
I am not sure on how this logic is any different from previous code. Its seems like previously code is also collecting podsCount per each node. Are we seeing any duplicate counts in the previous logic?

Reply:
From what I can see, with the previous code if you have 100 nodes, this will create 100 API calls... Where the code change that @rsevilla87 has will be a single API call no matter the node count then using the pod description he filters per-node.  Effectively doing the same thing, but with less API calls.

Reply:
Yes perfect. Make sense to me now.